lldb/include/lldb/API/SBPlatform.h

```diff
@@ -100,6 +99,9 @@ class LLDB_API SBPlatform {
 
   SBPlatform(const char *platform_name);
 
+  SBPlatform(const char *platform_name, const SBDebugger &debugger,
+             const char *script_name, const SBStructuredData &dict);
+
```

Comment:
is `script_name` the name of a class or a function? If it is a class that implements the platform API, then should we avoid passing in `platform_name` and have that name be discovered by a call to the platform class itself?

Reply:
That sounds like a good idea, I'll add a getter to the python base class.